airbyte-integrations/connectors/source-zoho-crm/source_zoho_crm/types.py

```diff
@@ -0,0 +1,223 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+import copy
+import dataclasses
+from decimal import Decimal
+from enum import Enum
+from typing import Any, Dict, Iterable, List, MutableMapping, Optional, Union
+
+import six
```

Comment:
my understanding is that `six` is used for python2/3 compatibility. Why are we using it here? we are only using py3